README.md

```diff
@@ -7,6 +7,44 @@
 > Automated JavaScript project management.
 
 
+## Migration from `v11`
+
+There was a large refactor from `v11` to `v12`, so there is some migration needed. All features you need should still be there, but you might need to changes some things.
+
+### Custom Config
+
+The `aegir` section in `package.json` was not really useful anymore so it is not supported anymore. Any custom configuration you need can be done in `.aegir.js`.
+
+### Pre and Post Hooks
+
+As we are no longer using `gulp` the previous setup using a custom `gulpfile` will not work anymore. To setup hooks around your tests you can use a new configuration option `hooks` in `.aegir.js`.
+
+```js
+// .aegir.js
+module.exports = {
+  hooks: {
+    pre (callback) {
+      console.log('I am called before node and browser tests')
+      callback()
+    },
+    post (callback) {
+      console.log('I am called after node and browser tests')
+      callback()
+    }
+  }
+}
+```
+
+You can also specify `hooks.browser` and `hooks.node` if you have a different setup for browser and/or node based tests.
```

Comment:
what about hooks.electron and hooks.cordova? Will it be easy to extend to support those runtimes as well?